bundles/framework/layerlist/components/LayerCollapse.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { List } from '../../../admin/admin-layereditor/components/List';
+import { Layer } from './LayerCollapse/Layer';
+import { Collapse, Panel } from '../../../admin/admin-layereditor/components/Collapse';
+import { Badge } from '../../../admin/admin-layereditor/components/Badge';
+import { Alert } from '../../../admin/admin-layereditor/components/Alert';
+
+const getLayers = (layers, otherProps) => layers.map(
+    (lyr, index) => {
+        const props = {
+            model: lyr,
+            even: index % 2 === 0,
+            ...otherProps
+        };
+        return <Layer key={index} {...props}/>;
```

Comment:
We should always use a real identifier as the key if possible. Layer id would be good here.
https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
Replaced with layer id